File: CustomChance/Net/View/ButtonStand.cs
using System;
using System.Drawing;
using SpaceVIL;
using SpaceVIL.Core;

namespace CustomChance
{
    internal class ButtonStand : ButtonCore
    {
        SpaceVIL.Rectangle _stand;

        public ButtonStand() : base() { }
        public ButtonStand(String text) : base(text) { }

        public override void InitElements()
        {
            base.InitElements();
            _stand = new SpaceVIL.Rectangle();
            _stand.SetBackground(Color.White);
            _stand.SetHeight(3);
            _stand.SetSizePolicy(SizePolicy.Expand, SizePolicy.Fixed);
            _stand.SetAlignment(ItemAlignment.Bottom | ItemAlignment.HCenter);
            _stand.SetVisible(false);
            AddItem(_stand);
        }

        public override void SetMouseHover(bool value)
        {
            base.SetMouseHover(value);
            if (value)
                _stand.SetVisible(true);
            else
                _stand.SetVisible(false);
        }
    }
}

File: CustomChance/Java/src/main/java/com/customchance/View/ButtonStand.java
package com.customchance.View;

import java.awt.Color;

import com.spvessel.spacevil.ButtonCore;
import com.spvessel.spacevil.Rectangle;
import com.spvessel.spacevil.Flags.SizePolicy;
import com.spvessel.spacevil.Flags.ItemAlignment;

class ButtonStand extends ButtonCore {
    Rectangle _stand;

    public ButtonStand() {
        super();
    }

    public ButtonStand(String text) {
        super(text);
    }

    @Override
    public void initElements() {
        super.initElements();
        _stand = new Rectangle();
        _stand.setBackground(Color.white);
        _stand.setHeight(3);
        _stand.setSizePolicy(SizePolicy.EXPAND, SizePolicy.FIXED);
        _stand.setAlignment(ItemAlignment.BOTTOM, ItemAlignment.HCENTER);
        _stand.setVisible(false);
        addItem(_stand);
    }

    @Override
    public void setMouseHover(boolean value) {
        super.setMouseHover(value);
        if (value)
            _stand.setVisible(true);
        else
            _stand.setVisible(false);
    }
}